cmd/kcp/kcp.go

```diff
@@ -147,5 +147,6 @@ func main() {
 
 	if err := cmd.Execute(); err != nil {
 		fmt.Fprintf(os.Stderr, "error: %v\n", err)
+		os.Exit(1)
```

Comment:
Could `k8s.io/component-base/cli#Run` be of help here?

Reply:
fixed